spec/order_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'order'
+require 'take_away'
+
+describe Order do
+
+  let(:take_away) { TakeAway.new }
```

Comment:
It's best to avoid relying on actual instances of classes other than the one under test and try to use doubles instead.

As it stands, if you were to continue working on this code base and accidentally introduced a bug into your takeaway class, it could prevent your tests here in the order class from running correctly, which could make bug fixing more difficult.